src/desktop/apps/artist2/server.js

```diff
@@ -14,6 +15,10 @@ app.get('/artist2*', adminOnly, async (req, res, next) => {
     const { ServerApp, redirect, status } = await buildServerApp({
       routes,
       url: req.url,
+      user: {
```

Comment:
We proabbly have to handle the case of logged-out better.

For instance, it might need to be `req.user && req.user.get('accessToken')`, as well as in Reaction, let's say you get back something like:
`user: { accessToken: undefined, id: undefined }`, that might get injected into metaphysics requests, which will lead to strange errors (since they shouldn't even be included, vs included with invalid values).